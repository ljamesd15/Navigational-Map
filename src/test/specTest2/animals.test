# Tests the specifications of loadGraph and findPath of GraphPaths.java using animals.
# Also tests to make sure it splits at underscores and reads them properly.

# Load the graph
LoadGraph animals animals_v1.tsv


# Find some paths
FindPath animals Dog Lion

FindPath animals Mouse Parrot

FindPath animals Mouse Lion

FindPath animals Leopard Dog

FindPath animals Parrot Parrot