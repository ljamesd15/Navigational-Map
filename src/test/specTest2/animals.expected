# Tests the specifications of loadGraph and findPath of GraphPaths.java using animals.
# Also tests to make sure it splits at underscores and reads them properly.

# Load the graph
loaded graph animals


# Find some paths
path from Dog to Lion:
Dog to Cat via House_Pet
Cat to Lion via Feline

path from Mouse to Parrot:
no path found

path from Mouse to Lion:
Mouse to Hamster via Rodent
Hamster to Cat via House_Pet
Cat to Lion via Feline

unknown character Leopard

path from Parrot to Parrot: